config/default/prometheus_servicemonitor_selector_patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: controller-manager-metrics-service
+  labels:
+    release: prom-azure-databricks-operator
+spec:
+  namespaceSelector:
+    matchNames:
+      - $(MetricsService.Namespace)
```

Comment:
I'm not 100% on this one but I think this maybe scraping everything in the namespace rather than just the operator. 

Looking at the docs I'd expect to see a `selector` with a `matchExpression` to limit it to only this operator. 

https://coreos.com/operators/prometheus/docs/latest/user-guides/running-exporters.html

Reply:
That patch file is used to get kustomize to perform variable substitution on the namespace selector in the actual `ServiceMonitor` definition (which has a `matchLabels` value defined to limit it to the operator :-)

Reply:
Another example of me not understanding kustomize - sorry!